assets/javascripts/omise-embedded-card.js

```diff
@@ -25,6 +25,12 @@ function showOmiseEmbeddedCardForm({
   }
   element.style.height = iframeElementHeight + 'px'
 
+  let fontName = font.name
+
+  if (font.name.trim() === OMISE_CUSTOM_FONT_OTHER && font.custom_name.trim() !== '') {
```

Comment:
What if I select Other, but do not put anything to the Custom font field?

Reply:
If that case ever occur then it will default to Poppins. We have added required attribute to the custom font name field if other is selected to avoid situation like this with HTML5 validation. You will see `Fill out this field` if you leave custom field empty.